plugin-build/src/main/kotlin/io/sentry/android/gradle/tasks/SentryUploadProguardMappingsTask.kt

```diff
@@ -87,4 +96,19 @@ abstract class SentryUploadProguardMappingsTask : Exec() {
         }
         return args
     }
+
+    companion object {
+        private const val PROPERTY_PREFIX = "io.sentry.ProguardUuids="
+
+        fun readUuidFromFile(file: File): String {
+            check(file.exists()) {
+                "UUID properties file is missing"
+            }
+            val content = file.readText().trim()
+            check(content.startsWith(PROPERTY_PREFIX)) {
+                "io.sentry.ProguardUuids property is missing"
+            }
+            return content.removePrefix(PROPERTY_PREFIX)
```

Comment:
another way of doing this would be via `java.util.Properties.load(InputStream inStream)`
not sure if Kotlin has a nicer extension

Reply:
do you think its worth changing it?

Reply:
Yeah I thought about using that as well, but I think that we probably don't need to use `java.util.Properties` as this is a file with a single property and we control it end-to-end (user is not supposed to edit it).

We could switch to `java.util.Properties` if we happen to use this file to pass more values.

Reply:
wfm, thanks